src/components/Network/Network.vue

```diff
@@ -104,32 +132,106 @@ export default {
       ],
       ingresses_headers: [
         {
-          text: 'Name',
+          text: 'Hosts',
+          value: 'hosts',
           algin: 'start',
-          sortable: false,
+          sortable: true
+        },
+        {
+          text: 'Status',
+          value: 'status'
+        },
+        {
+          text: 'Name',
           value: 'name'
         },
+        {
+          text: 'Annotations',
+          value: 'annotations'
+        },
         {
           text: 'TLS',
           value: 'tls'
         },
         {
-          text: 'Hosts',
-          value: 'hosts'
+          text: 'Paths',
+          value: 'path'
+        },
+        {
+          text: 'Service',
+          value: 'services'
+        },
+        {
+          text: 'Created at',
+          value: 'creation_timestamp'
         }
       ]
     }
   },
-  mounted () {
+  beforeMount () {
     if (this.services_data.length === 0) {
       this.$store.dispatch('services/request_services')
     }
     if (this.ingresses_data.length === 0) {
       this.$store.dispatch('ingresses/request_ingresses')
     }
+  },
+  mounted () {
+    this.check_available()
+  },
+  methods: {
+    check_availablity (data) {
+      this.ingress = data
+      data['status'] = data.rules.map(async rule => {
+        console.log('test route: ' + `${rule.host}`)
+        let statuscode
+        const XHR = new XMLHttpRequest()
```

Comment:
What does a XMLHttpRequest do? Couldn't we have just used our axios instance, via the backend.js file?

Reply:
no there are headers we don't need and want... I am really against axios inside components.